run_tests.py

```diff
@@ -14,7 +14,7 @@
     # args.extend(['--cov', 'xpdAcq'])
     # call pytest and exit with the return code from pytest so that
     # travis will fail correctly if tests fail
-    os.environ['IS_TEST'] = str(1)
+    os.environ['ENV_VAR'] = str(1)
```

Comment:
You might want to go with something more specific, this environmental variable could have been taken